server/views/pages/profile.html

```diff
@@ -61,7 +61,7 @@ <h1>{{ title }}</h1>
 {% block main %}
   <div class="govuk-body">
 
-{% if not signedInUser %}
+{% if not isSignedIn %}
```

Comment:
`signedInUser` really refers to the user's name. 

using new flag rather than presence of firstName